### PR TITLE
fix #68151: Allowed Edit Style window to remember last user resize

### PR DESCRIPTION
**Pull request by WASamK**

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
Ah, much better than #2461

File: mscore/editstyle.ui [hunk 1]
```diff
@@ -6,14 +6,29 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>1150</width>
-    <height>739</height>
+    <width>780</width>
+    <height>578</height>
    </rect>
   </property>
+  <property name="sizePolicy">
```
Comment (lasconic) [inline mscore/editstyle.ui]: Are these changes to the UI file really needed?

File: mscore/editstyle.cpp [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (lasconic) [inline mscore/editstyle.cpp]: Could you remove this line?

File: mscore/editstyle.h [hunk 3]
```diff
@@ -83,5 +86,3 @@ class EditStyle : public QDialog, private Ui::EditStyleBase {
 
 } // namespace Ms
 #endif
-
```
Comment (lasconic) [inline mscore/editstyle.h]: These two lines are fine. Please keep them.

**Comment by lasconic** [comment]
I asked for some changes. You can make then as follow.

```
$ git checkout WASamK:68151-editstyle

#Do the changes, you can get previous version of a file from github.com if necessary.

# check the changed files
$ git status
# check the diff of your changes 
$ git diff 

$ git add thefile.cpp thefile2.cpp
$ git commit -m "fix lines etc..."
$ git rebase -i HEAD~2
# change second line with squash see http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

$ git push --force origin WASamK:68151-editstyle

# this PR should now be updated with your changes and with a single commit.

```

**Comment by Jojo-Schmitz** [comment]
I'm doing this differently, 'git commit --amend`, just safe the commit message (changed or unchanged), undo last commit in GitHub for Windows, make the changes, commit again, and finally `git push -f`.

File: mscore/editstyle.ui (hunk 1 above)
Comment (WASamK) [inline mscore/editstyle.ui]: Without a single change to the original UI, the "remember last user resize" and "remember last user position" still works,
But the window can only be expanded. So all these changes are to allow the window to contract. But sure, will go through each one and see what is absolutely necessary.

**Comment by WASamK** [comment]
I accidently deleted all my commits. So I commited again and created another pull request. Hope that's okay.
https://github.com/musescore/MuseScore/pull/2500

**Pull request closed without merging**